webpack/assets/javascripts/react_app/components/SettingsTable/SettingsTableHelpers.js

```diff
@@ -111,19 +111,26 @@ const formatArraySelection = (attr, setting) => {
     return null;
   }
 
-  const [, model] = setting[attr].split('-');
-
-  const selectGroup = selectValues.find(group => group.groupLabel === model);
-
-  if (!selectGroup) {
-    return null;
-  }
-
-  const selectedItem = selectGroup.children.find(
-    item => item.value === setting[attr]
-  );
-
-  return selectedItem && selectedItem.label;
+  const findSelectValue = (attrValue, valuesSelection) => {
+    const foundItem = valuesSelection.reduce((memo, group) => {
+      if (memo) {
+        return memo;
+      }
```

Comment:
Why are you using a `reduce` here if don't modify the memo but just return it once you have something?